goal_weight: 1 # the weight determining how much the goal will affect the potential of a point
profile_diameter: 50 # n in the nxn square around the vessel for which potential is calculated
obstacle_weight: 100 # the weight determining how much the obstacles will affect the potential of a point
d_inf: 20 # the euclidean distance inside which obstacles affect the potential of a point
p_ngz: 50 # the potential added to a point inside the no-go zones
p_hyst: 10 # the extra potential added to a point behind the vessel i.e the coast of tacking
g_v: 10 # scalar multiplied to the velocity consideration
local_coord_scale: 100 # how many x,y there are per 100m i.e 100 = each x,y coordinate is one meter

default_waypoint_id: 0 # id for normal waypoints
waypoints_to_circle_id: 1 # id for waypoints that should be circled
waypoints_in_circle_id: 2 # id for waypoints in a circle
circle_time_limit: 180 # limit for circling a waypoint in sec

radius_earth: 6371 # radius of the earth in km
waypoint_radius: 10 # radius for waypoint in which the vessel should circle
num_circle_point: 8 # number of waypoints generated around a circle waypoint
